components/CatBoost/Predict_class_probabilities/from_CSV/component.py

```diff
@@ -57,5 +57,6 @@ def catboost_predict_class_probabilities(
         packages_to_install=['catboost==0.23'],
         annotations={
             "author": "Alexey Volkov <alexey.volkov@ark-kun.com>",
+            "canonical_location": "https://raw.githubusercontent.com/Ark-kun/pipeline_components/master/components/CatBoost/Predict_class_probabilities/from_CSV/component.py",
```

Comment:
for discussion, would it be better to use github.com urls?
I think they are more accessible, because you can easily find revisions, and sibling files

Reply:
I've thought about this, but decided on the Raw data URLs since they're more machine-accessible. So you can replace one `component.yaml` URL with another in `load_component_from_url`.

Reply:
fair enough, I think that convinced me